fe/fe-core/src/main/java/com/starrocks/sql/optimizer/transformer/WindowTransformer.java

```diff
@@ -246,8 +246,10 @@ public static List<WindowTransformer.PartitionGroup> reorderWindowOperator(
                         SqlToScalarOperatorTranslator.translate(analyticExpr, subOpt.getExpressionMapping());
                 ColumnRefOperator columnRefOperator =
                         columnRefFactory.create(agg.toString(), agg.getType(), agg.isNullable());
-                analyticCall.put(columnRefOperator, (CallOperator) agg);
-                subOpt.getExpressionMapping().put(analyticExpr, columnRefOperator);
+                if (agg instanceof CallOperator) {
```

Comment:
why need this if conditions? when one analyticExpr is call operator?

Reply:
Yes, if there are two identical analyticExprs, then the second expr will be `ColumnRefOperator`